modules/core/entity/src/OrganizationTypeManager.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace Drupal\farm_entity;
+
+use Drupal\Component\Plugin\Exception\PluginException;
+use Drupal\Core\Cache\CacheBackendInterface;
+use Drupal\Core\Extension\ModuleHandlerInterface;
+use Drupal\Core\Plugin\DefaultPluginManager;
+
+/**
+ * Manages discovery and instantiation of organization type plugins.
+ *
+ * @see \Drupal\farm_entity\Annotation\OrganizationType
+ * @see plugin_api
+ */
+class OrganizationTypeManager extends DefaultPluginManager {
+
+  /**
+   * Constructs a new )rganizationTypeManager object.
```

Comment:
```suggestion
   * Constructs a new OrganizationTypeManager object.
```

Reply:
Good eye! :eyes: I will amend my commit to fix this.